packages/frontend/src/lib/Navigation.svelte

```diff
@@ -69,6 +70,16 @@ onDestroy(() => {
     <SettingsNavItem icon={faRocket} title="Services" selected={meta.url === '/services'} href="/services" />
     <SettingsNavItem icon={faMessage} title="Playgrounds" selected={meta.url === '/playgrounds'} href="/playgrounds" />
 
+    <!-- Settings -->
+    <div class="pl-3 mt-2 ml-[4px]">
+      <span class="text-[color:var(--pd-secondary-nav-header-text)]">Server Information</span>
```

Comment:
here it's confusing, in the image I see another text: Configurable Options

Reply:
See https://github.com/containers/podman-desktop-extension-ai-lab/pull/2052#issuecomment-2459973798

Reply:
my point was that the image attached to this PR should be updated to reflect the code changes

cc @gastoner